.github/ISSUE_TEMPLATE/offboard-team-member.md

```diff
@@ -0,0 +1,82 @@
+
+### Departure checklist
+
+The manager should follow the checklist below to offpord a departing team member after exit. Don't hesitate to delegate some actions to others (e.g. if another person needs to give access to an account because you aren't able to).
+
+**First steps**
+
+- [ ] Schedule a 2i2c offboarding meeting
+- [ ] Arrange with CS&S HR team to do formal exit interview
+
+
+**Accounts**
+
+Document accounts:
+
+- [ ] Get member's [GitHub.com account](https://github.com)
+- [ ] Get member's [ReadTheDocs.org account](https://readthedocs.org)
+- [ ] Get member's [Quay.io account](https://quay.io)
+- [ ] Get member's [NameCheap.com account](https://namecheap.com)
+- [ ] Get member's [PagerDuty account](https://pagerduty.com)
+
+
+** After Departure**
+
+Remove 2i2c accounts and remove from team accounts:
+
+- [ ] Deactivate a @2i2c.org Google account for them
+	- Note that this requires [access to the Admin Panel](https://compass.2i2c.org/en/latest/administration/google-workspace.html?highlight=workspaces#access-and-permissions), a permission held by any 2i2c team lead
+- [ ] Remove them from the proper Google group in our Google Admin space.
+- [ ] Remove them from [2i2c GitHub teams](https://github.com/orgs/2i2c-org/teams/)
+- [ ] Remove them from [Slack groups](https://2i2c.slack.com/admin/user_groups)
+  - Note that as 2i2c slack is open check to see if they are going to continue to be part of broader 2i2c community and their slack email address is changed
+- [ ] Remove from the proper [FreshDesk group](https://2i2c.freshdesk.com/a/admin/groups)
+- [ ] Remove from [PagerDuty Users group](https://2i2c-org.pagerduty.com/users-new)
+- [ ] Remove from our [bitwarden organization](https://vault.bitwarden.com/#/organizations/11313781-4b83-41a3-9d35-afe200c8e9f1/vault) and remove from "Default collection"
+- [ ] Remove a team member from the Support Steward Team Role, calendar and rota [Role Offboarding Ticket](https://github.com/2i2c-org/team-compass/issues)
+
+
+**Documentation**
+
+- [ ] Revoke appropriate permissions on [ReadTheDocs projects](https://readthedocs.org/)
+  - [ ] https://readthedocs.org/dashboard/2i2c-pilot-hubs/users/
+  - [ ] https://readthedocs.org/dashboard/2i2c-team-compass/users/
+  - [ ] https://readthedocs.org/dashboard/2i2c-pilot-documentation/users/
+
+**Cloud engineering permissions**
+
+_This is only relevant for others that are leaving our Engineering team._
+
+- [ ] Remove from 2i2c hubs as an admin
+- [ ] Remove from appropriate GCP organizations and projects
+  - [ ] Organization `2i2c.org`'s [admin group](https://console.cloud.google.com/iam-admin/groups/03znysh73qbio4n?organizationId=184174754493)
+  - [ ] Organization `2i2c.org`s [engineering group](https://console.cloud.google.com/iam-admin/groups/01opuj5n2qnifml?organizationId=184174754493)
+  - [ ] Outside-organization project for [`cloudbank`](https://console.cloud.google.com/iam-admin/iam?project=cb-1003-1696)
```

Comment:
Longer term I feel like we should have a source of truth for all GCP/Azure/AWS non-2i2c projects/accounts since this will be out of date if we continue creating org-specific resources like this. But I think for now we should just go w/ this and iterate